example/package.json

```diff
@@ -18,6 +18,8 @@
     "react-native": "0.74.2",
     "react-native-reanimated": "3.13.0",
     "react-native-svg": "link:../",
+    "react-native-view-shot": "4.0.0-alpha.2",
+    "react-native-web": "^0.19.10",
```

Comment:
could we remove that react-native-web dependency?
We extract our web-example to another directory.

Reply:
Good point, I had to miss it while merging upstream changes. Will change.